packages/default/scss/dropdowntree.scss

```diff
@@ -0,0 +1,12 @@
+@import "variables";
+@import "mixins";
+
+
+// Dependencies
+@import "common/all";
+@import 'common/forms';
+
```

Comment:
It seems that the dropdowntree depends on the popup, so this should be listed as a dependency.